wandb/sdk/internal/sender.py

```diff
@@ -864,6 +864,9 @@ def _config_telemetry_update(self, config_dict: Dict[str, Any]) -> None:
         s = self._telemetry_obj.huggingface_version
         if s:
             config_dict[wandb_key]["huggingface_version"] = s
+        s = self._telemetry_obj.launch_run_queue_item_id
```

Comment:
probably a better solution is just to add an extra message that updates the internal config object of the run if you actually want to store it there

Reply:
Yes thats much nicer, do you think we can get away with sticking this in the config if I do it that way?

Reply:
Can i sneak the launch message into the telemetry proto file? @kptkin

Reply:
why do you need to put it in the telemetry proto? just put it in the `config["_wandb"]` with the launch namespaced (if you have to put it in the config)